jck/build.xml

```diff
@@ -39,7 +39,7 @@
 		<isset property="env.ADOPTOPENJDK_SYSTEMTEST_BRANCH"/>
 	</condition>
 
-	<condition property="stf_repo" value="${env.STF_REPO}" else="${git-prefix}://github.com/adoptium/stf.git">
+	<condition property="stf_repo" value="${env.STF_REPO}" else="${git-prefix}://github.com/adoptium/STF.gif">
```

Comment:
It should be `git`, not `gif`. The same for the rest of the files.

Reply:
Oops, sorry, I'll fix that right away.

Reply:
Fixed.

Reply:
Thanks :)